sass/_camper_blogs.scss

```diff
@@ -0,0 +1,42 @@
+.camper_blogs{
+  //background-color: $fccred;
+  width: 100%;
+  min-height: 200px;
+  padding: 5px 100px 25px 100px;
+  
+  h2{
+    text-align: center;
+  }
+  
+  .blog {
+    width: 600px;
+    margin: 10px auto;
+    background-color: white;
+    border-radius: 3px;
+    border: solid black 2px;
+    
+    .blog_img_container{
+      display: inline-block;
+      width: 120px;
+      height: 120px;
+      padding: 10px;
+    }
+    
+    img {
+      height: 100%;
+      width: 100%;
+      margin: 0px;
+    }
+    
+    .blog_description{
+      display: inline-block;
+      width: 460px;
+      text-align: center;
+      
+      h3{
+        margin: 0px;
+        padding: 0px;
+      }
+    }
+  }
+}
```

Comment:
The syntax could be improved. The indentation makes it hard to read what is nested where and several of the selectors are not followed by a space.

Empty return at end of file.